odoo/addons/base/tests/test_float.py

```diff
@@ -160,9 +160,9 @@ def try_roundtrip(value, expected, date):
 
         # res.currency.rate uses 6 digits of precision by default
         try_roundtrip(2.6748955, 2.674896, '2000-01-01')
-        try_roundtrip(-2.6748955, -2.674896, '2000-01-02')
+        try_roundtrip(0.6748955, 0.674896, '2000-01-02')
```

Comment:
@rco-odoo we want to remove the possibility to have negative rates in currencies (meaningless)... So you might want to use another object in that test? or can we just remove the lines testing negative values?

Reply:
as discussed: we'll comment those lines and put a TODO statement for that them being re-implemented with proper dedicated model.